Userland/Applications/Browser/StorageWidget.cpp

```diff
@@ -130,7 +130,7 @@ void StorageWidget::clear_session_storage_entries()
 void StorageWidget::delete_cookie(Web::Cookie::Cookie cookie)
 {
     // Delete cookie by making its expiry time in the past.
-    cookie.expiry_time = Duration::from_seconds(0);
+    cookie.expiry_time = UnixDateTime::now_coarse() - Duration::from_seconds(60);
```

Comment:
Epoch is not necessarily in the past :) so I'm kind of fixing a bug here.

Reply:
I didn't expect `Duration` to be able to store negative durations, so I thought that `epoch()` == `earliest()`. All good then (although, one could replace this with `earliest()` instead then?).

Reply:
The internal offset is an i64, so it's totally capable of before-1970 time. (Also, it really oughta be; fun calendar shenanigans amirite?)

Reply:
To clarify the bug here: `epoch() == earliest()`, yes. However, imagine the user's time is currently a few minutes before the epoch. Then, setting the cookie to the epoch will not delete it for a few minutes, since the expiry time is still in the future. That may seem like an arbitrary scenario, but I've recently seen enough software die when you (accidentally or intentionally) set your time to briefly before the epoch. I don't think the argument "we're 50 years past the epoch, who cares" is valid; let's have this code work no matter what time it currently is. After all, as you can see with the above it's very easy to obtain a point in time that's *always* in the past, no matter what the current time is.